sys-libs/musl-nscd/musl-nscd-1.0.3.ebuild

```diff
@@ -22,8 +22,7 @@ SLOT="0"
 IUSE="minimal"
 
 DEPEND="
```

Comment:
This is broken as-is and we need to revbump for adding the blockers to RDEPEND too.